core/include/webview/detail/engine_base.hh

```diff
@@ -318,6 +319,19 @@ protected:
     }
   }
 
+  void dispatch_size_default(bool m_owns_window) {
```

Comment:
Note: In my Windows environment, when passing in an existing window, this function is called 7 times.

Reply:
See previous comment and latest PR. Should mitigate this.

Reply:
Also, `dispatch_size_default` will execute, but do nothing if `!m_owns_window`.

This helps keep code DRYer/cleaner from the calling side by avoiding repeated
```
if (m_owns_window) {
  dispatch_size_default();
};
```
conditional blocks.

From the calling side, it is implicit that setting the default size behaviour has something to do with `m_owns_window`, and in my opinion this is cleaner and more readable code.